### PR TITLE
Admin nav bar home button and move admin home page to appropriate controller

### PR DESCRIPTION
**Pull request by jesswinssinger**
@cfstiff and @annaliseko you guys were assigned this because I moved a lot of UI stuff for the home page into partials (all same code, just in different files for clarity). Just wanted to make you aware of it!

Nav bar home button is now routed correctly for admin/non-admin users.

I moved the admin home page actions all to the static_page controller because it makes sense to have it be the root path for admins (and makes no sense to have its actions in the draw periods controller lol). 
Since the drop down with empty rooms from the non-admin home page is so useful, I added it at the top right. It's a little awkward that it shifts all the links down when it opens, but for the sake of getting this done in time I just left it. Let me know if you find errors!

### CODE REVIEW

**Comment by nupurb** [comment]
Looks good to me!

File: app/controllers/draw_periods_controller.rb [hunk 1]
```diff
@@ -1,91 +1,35 @@
...
     def create
         @draw_period = DrawPeriod.new(draw_period_params)
         
         if @draw_period.save
             flash[:notice] = 'Draw Period was successfully created.'
-            redirect_to admin_home_path
+            redirect_to root_path
         else
-            render :admin_landing_page
+            render "static_pages/home"
```
Comment (g-rocket) [inline app/controllers/draw_periods_controller.rb]: Can we user a `something_path` variable here?

File: app/controllers/draw_periods_controller.rb [hunk 2]
```diff
@@ -1,91 +1,35 @@
...
     end
 
     def update
         if @draw_period.update(draw_period_params)
             flash[:notice] = 'Draw Period was successfully updated.'
-            redirect_to admin_home_path
+            redirect_to root_path
         else
-            render :admin_landing_page
+            render "static_pages/home"
```
Comment (g-rocket) [inline app/controllers/draw_periods_controller.rb]: ...path?

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -1,174 +1,18 @@
...
-    There are approximately <%= pullable_rooms_number(dorm) %> pullable rooms in West. </br>
-<% end %>
-</div>
-
-
 
+<% if current_user == nil %>
+    <%= render partial: "logged_out_home" %>
+<% elsif current_user.is_admin %>
+    <%= render "static_pages/admin_home" %>
```
Comment (g-rocket) [inline app/views/static_pages/home.html.erb]: Admins who are students will need to be able to see the student homepage to pull their own room, I'd think?

Comment (g-rocket) [reply app/views/static_pages/home.html.erb]: Never mind; see slack discussion

**Pull request merged**